static/scripts/pay.ts

```diff
@@ -73,7 +98,7 @@ export const pay = async (): Promise<void> => {
       }
       await withdraw(signer, txData, predefined);
     } catch (error: unknown) {
-      console.error(error);
+      console.log(error);
```

Comment:
It doesn't make sense to me to change this to `console.log` if you are displaying an error you should use `console.error` (assuming you want to continue function execution. Normally you should just let the error throw normally without the try/catch.